### ISSUE TITLE
online validator

### ISSUE THREAD

**Issue by fgnievinski** [citation-file-format/citation-file-format#370]
It'd be nice to have online validator, where you'd just upload the cff file.
The existing validators require installation:
https://github.com/citation-file-format/citation-file-format#tools-to-work-with-citationcff-files-wrench

**Comment by fgnievinski** [comment]
an existing workaround is to create a repository on github and upload the cff file there, which will show warnings/errors when editing the file contents online.

**Comment by sdruskat** [comment]
Thanks, @fgnievinski, for making this suggestion!

Indeed, an online validator would be nice. I believe this may be something that is  already on the roadmap for [cffinit](https://github.com/citation-file-format/cff-initializer-javascript). @fdiblen, is this true?

**Comment by sdruskat** [comment]
Alternatively, you can also use the [cffconvert GitHub Action](https://github.com/marketplace/actions/cffconvert) to validate your file during CI. This doesn't require local installation either.

**Comment by dokempf** [comment]
FWIW with the CFF schema being on schemastore, generic online validation tools can also be used. This one can export links with the schema being hard-coded: https://www.jsonschemavalidator.net/s/anOx05M3

**Comment by sdruskat** [comment]
> FWIW with the CFF schema being on schemastore, generic online validation tools can also be used. This one can export links with the schema being hard-coded: https://www.jsonschemavalidator.net/s/anOx05M3

Indeed, thanks for the heads up. The schemastore.org inclusion of CFF also means that you can use it directly in some IDEs/editors (see https://www.schemastore.org for a list) for auto-completion and validation.

**Comment by abelsiqueira** [comment]
Just passing to say that on cffinit 2.2.0 you can now update your existing CFF, so it can act as a validator.
It will forcefully update the `cff-version` to 1.2.0, so there might be errors from things that were previously overlooked, but that's probably a good thing.
Direct link: https://citation-file-format.github.io/cff-initializer-javascript/#/update

**Comment by sdruskat** [comment]
It would probably be a good thing to promote online validation via cffinit on the CFF website.
